src/content/en/fundamentals/media/video-preload.md

```diff
@@ -0,0 +1,399 @@
+  });
+</script>
+```
+
+With `preload` value set to `auto`, we're saying that downloading the entire
+video right away is considered desirable.
+
+```
+<video id="video" preload="auto" src="file.mp4" controls></video>
+
+<script>
+  video.addEventListener('progress', function() {
+    if (video.buffered.length === 0) return;
+
+    var bufferedSeconds = video.buffered.end(0) - video.buffered.start(0);
+    console.log(bufferedSeconds + ' seconds of video are ready to play!');
+  });
+</script>
+```
+
+There are some caveats though. As this is just a hint, browser may completely
+ignore the `preload` attribute. At the time of writing, here are some rules
+applied in Chrome:
+
+- When [Data Saver]{: .external} is enabled, Chrome forces `preload` value to
+  `none` if video URL starts with "http" or "https".
+- In Android 4.3, Chrome forces `preload` value to `none`.
+- On a cellular connection (2G, 3G, and 4G), Chrome forces `preload` value to
+  `metadata`.
+
+## Link preload
+
+As [covered] in other [articles]{: .external }, [link preload]{: .external} is
+a declarative fetch that allows you to force the browser to make a request for
+a resource without blocking the document's `onload` event. Resources loaded via
+`<link rel="preload">` are stored locally in the browser, and are effectively
+inert until they're explicitely referenced in the DOM, JavaScript, or CSS. 
+
+Preload is different from prefetch in that it focuses on current navigation and
+fetches resources with priority based on their type (script, style, font,
+video, audio, etc.). It should be used to warm up the browser cache for current
+sessions.
+
+<figure class="TODO">
+  <img src="/web/fundamentals/media/images/video-preload/link-preload.png">
+</figure>
+
+### Preload full video
+
+Here's how to use it to preload a full video  on your website so that
+when your JavaScript asks to fetch video content, it is read from disk cache as
+the resource may have already been cached by the browser. If preload request
+didn't finish yet, a regular network fetch will happen. I'd recommend using
+this only for audio songs or short video clips.
+
+```
+<link rel="preload" as="video" href="https://cdn.com/file.mp4" crossorigin>
+
+<video id="video" controls></video>
+
+<script>
+  // Later on, after some condition has been met, set video source to the
+  // preloaded video URL.
+  video.src = 'https://cdn.com/file.mp4';
+  video.play().then(_ => {
+    // If preloaded video URL was already cached, playback started immediately.
+  });
+</script>
+```
+
+As the preloaded resource is going to be consumed by a video element in the
+example, the `as` preload link value is `video`. If it were an audio element,
+it would be `as="audio"`.
+
+If you are preloading links with CORS enabled resources you must also include
+the `crossorigin` attribute in order for the resource to be properly used.
+
+### Preload first segment
+
+Here's another example below of how to preload the first segment of a video
```

Comment:
Either:

'Here's another example of how...'
_or_
'The example below shows how...'

I prefer the second.